srcpkgs/fail2ban/template

```diff
@@ -10,12 +10,13 @@ maintainer="necrophcodr <necrophcodr@necrophcodr.me>"
 license="GPL-2.0-only"
 homepage="http://www.fail2ban.org/"
 distfiles="https://github.com/${pkgname}/${pkgname}/archive/${version}.tar.gz"
```

Comment:
Please expand `pkgname` here to `fail2ban`.

Reply:
Oh, ok. I left is as it was because I assumed it's the preferred way.